fullstack/pom.xml

```diff
@@ -24,6 +24,7 @@
         <awaitility.version>1.7.0</awaitility.version>
         <commons-io.version>2.5</commons-io.version>
         <commons-lang.version>3.4</commons-lang.version>
+        <ehcache3.version>3.2.0</ehcache3.version>
```

Comment:
let's wait for spring boot 1.5.0 to be released : https://github.com/spring-projects/spring-boot/blob/1.5.x/spring-boot-dependencies/pom.xml#L74